packages/arb-token-bridge-ui/src/util/withdrawals/helpers.ts

```diff
@@ -23,6 +23,10 @@ import { fetchNativeCurrency } from '../../hooks/useNativeCurrency'
 export type EthWithdrawal = L2ToL1EventResult & {
   l2TxHash?: string
   transactionHash?: string
+  direction: 'deposit' | 'withdrawal'
```

Comment:
it's `EthWithdrawal` so I'd expect the direction to be `withdrawal` only?

Reply:
Updated